Source/WebCore/layout/LayoutUnits.h

```diff
@@ -52,6 +52,8 @@ using InlineLayoutRect = LayoutRect;
 
 struct Position {
     operator LayoutUnit() const { return value; }
+    friend bool operator==(const Position&, const Position&) = default;
```

Comment:
I can’t figure out what rule you are using to decide when to use `const&` and when to just pass the structure. I would have expected just `Position` here, but I’m not sure it matters.

Reply:
I'll switch to `Position`, I didn't realize the type was so small.

Reply:
There are others like this even in this patch, such as `LayoutUnit` itself. I didn’t try to thoroughly search for such things.